examples/linearStaticCantileverAero/dragCoefFunction.m

```diff
@@ -1,8 +1,11 @@
 function C_d = dragCoefFunction (betaRel)
     % Aeroelastic stability of a 3DOF system based on quasi-steady theory with reference to inertial coupling
     % Emulation of drag extrated from the reference above
-    if max(betaRel)< deg2rad(210)
-        omega_drag = 0.8 * 2 * pi / deg2rad(180);
-        C_d = +1.2 - 0.6* cos( omega_drag * betaRel);
+    if 0<betaRel< deg2rad(210)
+      omega_drag = 0.8 * 2 * pi / deg2rad(180);
+      C_d = +1.2 - 0.6* cos( omega_drag * betaRel);
+    else 
+      betaRel
+      error ("The angle must be beteween 0º and 210 to use this aerodynamic coefficents")
```

Comment:
```suggestion
      error ('The angle must be between 0 and 210 degrees to use these aerodynamic coefficients')
```
corrijo between, these y coefficients ...
también uso comillas simples para que funcione también en matlab
también quito el símbolo de grado y lo pongo en palabra...

Reply:
Done, didin't know that " didn't work using Matlab. I will start using it